### PR TITLE
Added danish support to locales.json

### PR DESCRIPTION
**Pull request by ItzLue**

### CODE REVIEW

File: config/locales.json [hunk 1]
```diff
@@ -76,5 +76,17 @@
     "locale": "Vietnamese",
     "Đ": "D",
     "đ": "d"
+  },
+  "da": {
+    "locale": "Danish",
+    "Æ": "AE",
+    "Ø": "OE",
+    "Å": "AA",
```
Comment (simov) [inline config/locales.json:84]: Can you add the lower case letters as well? Also `Æ` seems to be part of the main charmap with correct transliteration in your case.

Comment (ItzLue) [reply config/locales.json]: Yes, thank you.

**Comment by simov** [comment]
Published in v1.6.1

**Comment by wartab** [comment]
Should the slugified version of ">" not be "storre end" instead of "større end"?

**Pull request merged**